apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1253 @@
+                              }
+                              onCheck={(e) =>
+                                handleComparisonChange({
+                                  id: dataItem.id,
+                                  nameIs:
+                                    locale === 'en'
+                                      ? dataItem.nameEn
+                                      : dataItem.nameIs,
+                                  iconSrc: universities.filter(
+                                    (x) => x.id === dataItem.universityId,
+                                  )[0].logoUrl,
+                                })
+                              }
+                              checked={
+                                selectedComparison.filter(
+                                  (x) => x.id === dataItem.id,
+                                ).length > 0
+                              }
+                              buttonLabel={n('apply', 'Sækja um')}
+                              checkboxLabel={n('compare', 'Setja í samanburð')}
+                              checkboxId={dataItem.id}
+                              cta={createPrimaryCTA(dataItem)}
+                              href={
+                                linkResolver('universitysearchdetails', [
+                                  dataItem.id,
+                                ]).href
+                              }
+                              infoItems={[
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'school'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n(
+                                    dataItem.degreeType,
+                                    TranslationDefaults[dataItem.degreeType],
+                                  )}`,
+                                },
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'calendar'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n('begins', 'Hefst')} ${n(
+                                    dataItem.startingSemesterSeason,
+                                    TranslationDefaults[
+                                      dataItem.startingSemesterSeason
+                                    ],
+                                  )} ${dataItem.startingSemesterYear}`,
+                                },
+                                // {
+                                //   icon: (
+                                //     <Icon
```

Comment:
I think this should rather be introduced once the data is available, instead of having this much code commented out all over the place. Im not sure, though, what the overall practice is for the web. What do you say @RunarVestmann?

Reply:
Perhaps possible to keep this PR as a draft, while its being worked on and test as a feature branch? 
Otherwise, some of this functionality needs to be broken down and shipped in smaller pieces.